### ISSUE TITLE
how to get proper save.json and data into the bot?

### ISSUE THREAD

**Issue by Mikunics** [MizaGBF/Rosetta-Public#2]
Hello, I've tried self hosting the bot for our discord server as well. I was able to make a dummy json save file after tinkering around but it seemed to be problematic as a bunch of other features did not seem to work when using the bot like gbf profile set or grabbing GW.sql. If its possible to ask, how would one go around properly setting up the data needed for the bot?

**Comment by MizaGBF** [comment]
Admin commands exist to edit/set up most things (in the Owner cog, which isn't documented in the help, you'll have to do `/owner` in discord to see what's available or open ~~`owner.py`~~ `admin.py` and read the descriptions of each command). It's possible I forgot something tho.
I won't give proper help about the account functionality itself, as I already mentionned on the previous bot [here](https://github.com/MizaGBF/MizaBOT/issues/8) but it should be easy enough to figure out on your own.
  
For the sql files, they are private. I guess I could start considering making them available somewhere, or the latest one at least, I'll think about it. (EDIT: and Github doesn't seem to allow uploading those in this reply.)
Speaking of which, I did modifications on that regard for the upcoming GW, so there is a small chance it will break (but I'll make sure to patch it quickly).

**Comment by MizaGBF** [comment]
Correction: in the `admin.py` cog, not `owner.py`.

**Comment by MizaGBF** [comment]
Here's a [folder](https://drive.google.com/drive/folders/11DcUKeO6Szd5ZEJN9q57MQl772v64_R2?usp=sharing) with past sql files.
I recommend just grabbing the last one (GW71), rename it to GW.sql and put it in the drive folder used for files (not the save one).

I added a bit more infos at the bottom of the readme, I figured out more people might run into similar troubles.  

If nothing else troubles you, feel free to close the issue.
